libs/@guardian/libs/src/measure/measure.ts

```diff
@@ -0,0 +1,49 @@
+import { isNonNullable } from '../isNonNullable/isNonNullable';
+import type { TeamName } from '../logger/@types/logger';
+import type { MeasureEnd as EndMeasure } from './@types/measure';
+
+/** For browser which do not fully support the performance API */
+const fallback = () => {
+	return -1;
+};
+
+/** Serialisation is an implementation detail */
+const SEPARATOR = ':';
+
+/**
+ * Helper to measure the duration between two events.
+ *
+ * Once ended, measures are appended to the list of `PerformanceMeasure`
+ * and can be retrieved with `performance.getEntriesByType('measure')`
+ *
+ * @example
+ * const stopMeasure = startMeasure('dotcom', 'fetch');
+ * await fetch('https://www.theguardian.com/uk.json');
+ * const duration = stopMeasure();
+ */
+export const startMeasure = (
```

Comment:
I wonder what’s the clearest name, here:

- `startMeasure`
- `startPerformanceMeasure`
- `startTimer`
- `start`

cc. @sndrs

Reply:
I like aligning the naming to the API so I'd vote for `startMeasure`

Reply:
the only thing with this is that 'measure' can refer to more than time (e.g. dimensions). `startPerformanceMeasure` read pretty unambiguously to me?